.readthedocs.yml

```diff
@@ -1,11 +1,15 @@
-version: 2
```

Comment:
It would have been fine to keep `version: 2` at the top of the file.

Reply:
Saw it at the bottom in the other repo, wanted to keep it consistent